UI/window-basic-main.cpp

```diff
@@ -557,6 +557,16 @@ void OBSBasic::UpdateVolumeControlsPeakMeterType()
 	}
 }
 
+void OBSBasic::UpdateVolumeControlsUpdateRate()
+{
+	double meterUpdateRate =
+		config_get_uint(basicConfig, "Audio", "MeterUpdateRate");
+
+	for (size_t i = 0; i < volumes.size(); i++) {
```

Comment:
You can omit the braces around this `for` loop because it only contains a single statement.

Reply:
I was following existing code style here again (https://github.com/obsproject/obs-studio/blob/master/UI/window-basic-main.cpp#L532-L534). Pretty easy to change if we don't mind having inconsistent styling.

Reply:
The [style guide we link to](https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst) says to omit the braces for non-function statement blocks that only contain a single line.  The only reasons they aren't omitted in certain places is because clang-format does not catch it, and Jim doesn't enforce it.

Reply:
Fair enough, fixed.